src/bpfbytecode.cpp

```diff
@@ -186,8 +186,11 @@ void BpfBytecode::load_progs(const RequiredResources &resources,
     bpf_program__set_log_buf(prog.bpf_prog(), log_buf.data(), log_buf.size());
   }
 
+  std::vector<Probe> special_probes;
+  for (auto probe : resources.special_probes)
+    special_probes.push_back(probe.second);
```

Comment:
Not very nice that we have to copy here but I don't have a better solution at the moment.